dbl-works/pdf-renderer
# Update dependencies & Dockerfile

Tested locally:
* build docker image ✅ 
* run docker image ✅ 
* send HTML to the endpoint ✅ 
* verify the returned Base64 string is a valid PDF ✅ 

#### After merge
- [ ] create release `v1.2.0`
- [ ] push new tag to Dockerhub
